plugins/modules/kutils.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Alexander Bakanovskii <skottttt228@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: kutils
+short_description: Kerberos utils for managing tickets
+description:
+  - Manage kerberos tickets with kinit, klist and kdestroy base utulities.
+author: "Alexander Bakanovskii (@abakanovskii)"
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+options:
+  password:
+    description:
+      - Principal password.
+      - It is required to specify O(password) or O(keytab_path).
+    type: str
+  principal:
+    description:
+      - The principal name.
+      - If not set, the user running this module will be used.
+    type: str
+  state:
+    description:
+      - The state of the kerberos ticket.
```

Comment:
Maybe? @felixfontein you usually have sharper eyes for these things, should we capitalize Kerberos mentions?
```suggestion
      - The state of the Kerberos ticket.
```